README.rst

```diff
@@ -56,6 +57,8 @@ in search mode is provided below:
         bind_dn: "cn=hacker,ou=svcaccts,dc=example,dc=com"
         bind_password: "ch33kym0nk3y"
         #filter: "(objectClass=posixAccount)"
+        # Alternatively, simple binding with a custom bind pattern:
+        #bind_pattern: "WORKGROUP\\%s"
```

Comment:
putting this here suggests it only works in `search` mode, whereas the opposite appears to be true?

Reply:
also: I suggest the use of single-quotes to avoid the double-backslash:
```suggestion
        #bind_pattern: 'WORKGROUP\%s'
```